step-certificates/values.yaml

```diff
@@ -9,18 +13,53 @@ nameOverride: ""
 # fullnameOverride overrides the full name of the chart.
 fullnameOverride: ""
 
+securityContext:
+  runAsUser: 1000
+  runAsGroup: 1000
+  fsGroup: 1000
+
 # image contains the docker image for step-certificates.
 image:
-  repository: smallstep/step-ca
-  tag: 0.15.6
+  repository: cr.step.sm/smallstep/step-ca
+  initContainerRepository: busybox:latest
   pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+registrationAuthority: false
+
+# Use existing secrets instead of 'bootstrap' init-container or 'inject'-element from helm chart
+# Note, the MutatingWebhookConfiguration created by autocert is not patched with CA bundle as the bootstrap init-container is not run
+existingSecrets: 
+  enabled: false
+  ca: false
+  issuer: false
+  configAsSecret: false
+  sshHostCa: false
+  sshUserCa: false
+  #generate can help you generate your secret on first install. It configures step-ca as a registration authority and connect it to vault.
+  generate:
+    configAsSecret: 
+      enabled: false
+      provisioners: ""
+      vault:
+        certificateAuthority: http://vault.vault.svc.cluster.local:8200
+        certificateAuthorityFingerprint: ""
+        pkiMountPath: pki_int1
+        pkiRoleRSA: "rsa-role"
+        pkiRoleEC: "ec-role"
+        pkiRoleEd25519: "ed25519-role"
+        authType: "kubernetes"
+        role: "smallstep"
```

Comment:
Finally these are the last changes that I noticed, as I mention before most of those fields should be in the `inject` block after. And disabled by default.